### DISCUSSION TITLE
Anschlusseinstellungen rote Anzeige bei Display

### DISCUSSION THREAD

**Discussion by jblond4711** [tbnobody/OpenDTU#712] [General]
Hi,

hat mal jemand einen Tipp wie ich die Anzeige die rot markiert ist wieder wegbekomme?
In älteren Firmware Versionen gab es die roten Stellen nicht. 

Edit: Der Blitz im Display fehlt auch.

![96524377-F2B9-4A66-A10C-22881453CA7E](https://user-images.githubusercontent.com/17287119/226099509-edbba46b-8d57-4ea0-9a9d-4e82465d88ef.jpeg)

**Comment by tbnobody** [comment]
Das Rot in der Zelle zeigt einfach an, dass Ausgewählt und Aktiv unterschiedlich sind. Ich baue ggf. noch etwas ein, damit "leer" und 255 bzw. -1 als identisch dargestellt werden.

> Edit: Der Blitz im Display fehlt auch.

Jeb. Der ist weg. Bringt keine Mehrinformation.

**Comment by jblond4711** [comment]
Hi,

eben habe ich vorsichtshalber nochmal komplett alles neu installiert.
Würde es nicht mehr Sinn machen die Werte die jetzt rot sind wegzulassen? Bei den Oled Displays gibt es doch gar keinen „cs“ und „reset“ Anschluss, oder bin ich nur mal wieder zu blond das zu verstehen?

**Comment by jblond4711** [comment]
Hi,

auch mit der aktuellen Firmware werden die roten Balken weiterhin angezeigt.
Weshalb gibt es denn die „cs“ und „reset“ Anzeige beim SH1106 Display? Kann man die nicht komplett entfernen?

**Reply by tbnobody** [reply]
> Kann man die nicht komplett entfernen?

Nein kann man nicht. Die stehen nämlich auf deaktiviert und auch das möchte man ja ggf. sehen. 
Drück mal Strg+F5 im Browser. Dann entfallen die roten Markierungen für die Pins die -1, 255 oder leer sind.
